src/v3/preact.config.js

```diff
@@ -58,6 +59,20 @@ export default {
       (plugin) => !(plugin instanceof MiniCssExtractPlugin),
     );
 
+    if (env === 'production') {
+      config.plugins.push(
+        new DefinePlugin({
+          DEBUG: false
+        })
+      );
+    } else {
+      config.plugins.push(
+        new DefinePlugin({
+          DEBUG: true
+        })
+      );
+    }
```

Comment:
Had to define this value because v2 `Logger` relies on it.

Reply:
possible to just add variable like this for development env? if possible we should avoid unnecessary global variables in the prod bundle

Reply:
In v2, `DEBUG` variable is set in the webpack.common config file. If we only set it for non production environments, we will get a js error because it is referencing a variable that is undefined. 

You can see how it is used here: `src/util/Logger.js` on line 16.

Reply:
You can find the DefinePlugin for v2 here: `scripts/buildtools/webpack/plugins.js`

Reply:
`DEBUG` means `window.DEBUG`, not sure why it causes error in prod bundle if it's not added. So you tested against the prod bundle?

Reply:
Oh no, I definitely tested against the dev bundle, not sure why it throws the error if it is truly `window.DEBUG`. But until I replicated that plugin from v2, it threw that error and wouldnt let me proceed.

Reply:
I added a comment to this code, the DefinePlugin only finds and replaces variables in the code. I confirmed it is needed for both dev and prod because if this isnt there the app wont load because the variable is unresolved. Added a link to the webpack docs as well.